src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,18 @@
+import { Statuses } from '../types/Statuses';
+import { Todo } from '../types/Todo';
+
+interface Filter {
+  status: Statuses;
+}
+
+export const getFilteredTodos = (todos: Todo[], { status }: Filter) => {
+  let filteredTodos = [...todos];
```

Comment:
In this case, you don't need let. 
You can use const and return result of action with array.